sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1805,10 +1805,37 @@ def hpat_pandas_series_corr_impl(self, other, method='pearson', min_periods=None
 @sdc_overload_method(SeriesType, 'head')
 def hpat_pandas_series_head(self, n=5):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.div
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/series_head.py
+       :language: python
+       :lines: 27-
+       :caption: Getting the first n rows.
+       :name: ex_series_head
+
+    .. code-block:: console
+
+        > python ./series/series_head.py
+        0    7
+        2    6
+        4    5
+        dtype: int64
```

Comment:
Let's replace such block with another one everywhere:
```
    .. command-output:: python ./series/series_head.py
       :cwd: ../../../examples
```

It should insert the result of the examples to the documentation automatically.